include/nrm.h

```diff
@@ -73,8 +73,23 @@ int nrm_finalize(void);
 #define NRM_LOG_INFO 4
 #define NRM_LOG_DEBUG 5
 
+/**
+ * Initializes NRM logging
+ *
+ * @param f: file descriptor
+ * @param nm: prefixed name for output
```

Comment:
`nm` is for `namespace`, meaning the component within nrm or the application that the logs are coming from.